news/2020-19-11-mappers-report-september-october.md

```diff
@@ -0,0 +1,213 @@
+
+And don't forget. Should you have any interesting contests planned or know about, let us know! We would love to give them more attention in our news post!
+
+### PDC2020
+
+Good news! The classic **Pending Cup** mapping contest is back for another round and currently accepting submissions! Time is tight though, as the current deadline for submission is **November 8th**. If you were thinking of entering but forgot to submit, take this as a reminder to do so - and if this is your first time hearing about it, maybe try it out and see how it goes! The Pending Cup has always been a source of awesome maps each year, and we expect this year will be no different. So even if you don't want to enter, be sure to stay tuned to see who comes out on top! You can find detailed information about PDC 2020 in its [dedicated forum thread](https://osu.ppy.sh/community/forums/topics/1155108).
+
+### Monthly Beatmapping Contest
+
+The Monthly Beatmapping Contest series is continuing as strongly as ever, albeit with some changes, but also with some awesome results to share! Starting with MBC \#8, the contest has changed to a judge-based format. Where before screeners would pick maps for the community to vote on, now screeners function the same but the final vote comes down to a panel of judges with detailed impressions and scoring. For MBC \#9 the formula was further mixed up by making the contest have a **theme** instead of a **restriction**. With these changes, we get to see an ever-evolving regular contest format to stay excited about!
+
+For MBC \#8 the restriction was that sliders could not be used, and [Mao](https://osu.ppy.sh/users/2204515)'s entry of [Silentroom - NULCTRL](https://osu.ppy.sh/beatmapsets/1266008#osu/2631203) with its excellent use of circles, **circle size 7**, and screen-usage based patterning won over the judge's hearts. He has now earned enough points in MBC for a tier 2 profile badge, and is all that much closer to attaining the coveted title of elite mapper! Well deserved!
+
+In MBC \#9 the theme was that every object must overlap visible with another one. With its results mapper [xbopost](https://osu.ppy.sh/users/6842421) came in first place with his entry of [PUP - See You At Your Funeral](https://osu.ppy.sh/beatmapsets/1289138#osu/2676185) which took the theme well to heart and built a unique and interesting map and gameplay experience that you should be sure to check out! That's not all though, as this is xboposts' **third** time coming in first place in MBC and they have now earned the esteemed title of **Elite Mapper**! It's certainly well deserved, with each of their entries providing well built and interesting experiences that make excellent use of the challenges MBC provides. They are certainly a mapper to keep up with in the future as well!
```

Comment:
"visible" in first sentence should be deleted?
also last 2 sentences seem like they should be on a new line?